lib/middleware.js

```diff
@@ -512,6 +516,22 @@ module.exports = function middlewareConstructor(env) {
           res.json({"status": 200});
         });
       };
+    },
+
+    /**
+     * If there's an oauth token, decrypt it and set the
+     * local user object.
+     */
+    maybeDecryptToken: function(req, res, next) {
```

Comment:
yes: name it after what you're guaranteeing is the state at the next middleware. *why* do you need either "no token" or a decrypted token? name the function after that need